tripgo.swagger.yaml

```diff
@@ -2506,6 +2723,73 @@ definitions:
       urls:
         - nuremberg-bv-de.hadron.buzzhives.com
 
+  RealtimeSnapshot:
+    description: Real Time Snapshot stats.
```

Comment:
we could only give matching numbers, and leave non-matching stuff away, or just say it has real time support, maybe say it contains alerts/positions/trip updates?